alkem/parser.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform ALKEM Digital Library
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  let match;
+
+  if (/^\/index\.php$/i.test(path) && param.view) {
```

Comment:
The condition `index.php + view param` looks a bit unrestrictive. Is it enough to discriminate book consultations without conflicting with other types of URL ? Restricting to `view = article` might be safer.

```suggestion
  if (/^\/index\.php$/i.test(path) && param.view === 'article') {
```

Reply:
made this change